### PR TITLE
Nerfs borg tipping's stun from 60 seconds to just 20 seconds

### PR DESCRIPTION
**Pull request by ReturnToZender**
## About The Pull Request
I love Silicons.

This PR makes it so instead of the agonizingly long 60 second timer on righting yourself as a borg, you have a 20 second timer instead. It's still a very long stun, but is not an entire minute in a game where you can be killed in 3 seconds.
See below for how long an entire minute stun is.

https://github.com/Bubberstation/Bubberstation/assets/110273561/4fb15f99-4816-4106-b388-a787635052bc

## Why It's Good For The Game
There's long, and then there's too long. This is too long, and makes the game worse for cyborg players without much tangible benefit.

## Proof Of Testing
With & without using beep emote
https://github.com/Bubberstation/Bubberstation/assets/110273561/f20c3640-46c4-4d86-a618-384db338dbc8

https://github.com/Bubberstation/Bubberstation/assets/110273561/e1f718f7-a5fe-4f0a-8d97-fd6ce0af6c52

## Changelog

:cl: ReturnToZender
balance: The stun time on the tippable component for borgs has been reduced to 20 seconds, from 60 seconds.
/:cl:

### CODE REVIEW

**Comment by ReturnToZender** [comment]
I included Robotic pleas for help and mercy in the footage to make the full 60 second stun more bearable. You're welcome, maintainers.

**Review by Swiftfeather — APPROVED** [review]
why not

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 1]
```diff
@@ -7,12 +7,13 @@
 	AddComponent(/datum/component/tippable, \
 		tip_time = 3 SECONDS, \
 		untip_time = 2 SECONDS, \
-		self_right_time = 60 SECONDS, \
+		self_right_time = 20 SECONDS, \
```
Comment (Swiftfeather) [inline code/modules/mob/living/silicon/robot/robot.dm:10]:
```suggestion
		self_right_time = 20 SECONDS, \ //BUBBER EDIT: REDUCES THE SELF-RIGHT TIME FOR BORGS TO 20 SECONDS, FROM 60 SECONDS
```

File: code/modules/mob/living/silicon/robot/robot.dm [hunk 2]
```diff
@@ -7,12 +7,13 @@
...
 		post_tipped_callback = CALLBACK(src, PROC_REF(after_tip_over)), \
 		post_untipped_callback = CALLBACK(src, PROC_REF(after_righted)), \
 		roleplay_friendly = TRUE, \
 		roleplay_emotes = list(/datum/emote/living/human/buzz, /datum/emote/living/human/buzz2, /datum/emote/living/beep, /datum/emote/living/human/beep2), /* SKYRAT EDIT CHANGE - ORIGINAL: roleplay_emotes = list(/datum/emote/silicon/buzz, /datum/emote/silicon/buzz2, /datum/emote/living/beep), */ \
 		roleplay_callback = CALLBACK(src, PROC_REF(untip_roleplay)))
+		//BUBBER EDIT: REDUCES THE SELF-RIGHT TIME FOR BORGS TO 20 SECONDS, FROM 60 SECONDS
```
Comment (Swiftfeather) [inline code/modules/mob/living/silicon/robot/robot.dm:16]:
```suggestion
```

**Review by Swiftfeather — CHANGES_REQUESTED** [review]

**Review by Swiftfeather — APPROVED** [review]

**Comment by Swiftfeather** [comment]
OH THAT'S WHY YOU DID IT SO SILLY. Alright oops

**Pull request merged**